Tests/C++WorkshopTests/CardsTests.mm

```diff
@@ -53,7 +53,15 @@ auto draw() -> Card<Deck>& {
 auto haerin = PMCard{"Haerin, COVID Chef", 2298, 4594};
 auto wen = PMCard{"Wen, Global Nomad", 427, 4140};
 
-auto standard_deck = Deck{{ mario, ken, steven, adrian, aleks, cindy, fletch, eitan, haerin }};
+auto make_unique(std::vector<Card> cards) {
```

Comment:
Having a function name that is the same as a `std` function with similar intention seems a bit confusing?
How do you feel about `create_unique` or something?

Reply:
My thought here was that this is pretty closely analogous to the standard version, so it could be (kind of) an overload of it.